c7n/resources/elasticache.py

```diff
@@ -458,3 +458,16 @@ def _cluster_eligible_for_snapshot(cluster):
         cluster['Engine'] != 'memcached' and not
         TTYPE.match(cluster['CacheNodeType'])
     )
+
+
+@resources.register('elasticache-group')
+class ElastiCacheReplicationGroup(QueryResourceManager):
+
+    class resource_type(TypeInfo):
+        service = "elasticache"
+        enum_spec = ('describe_replication_groups',
+                     'ReplicationGroups[]', None)
+        arn_type = 'replicationgroup'
+        id = name = dimension = 'ReplicationGroupId'
+
```

Comment:
is this supported by resource group tagging? if so it should have the augment and tag actions as well

Reply:
Last I checked replication group tagging was not supported

Reply:
I might be wrong!! Double checking, thanks!

Reply:
The api/console doesn't let me tag a replication group! Replication group by itself doesn't expose any tags as per my findings! Also, I created a replication group using a cli where I could specify tags as input as seen in this documentation: https://docs.aws.amazon.com/cli/latest/reference/elasticache/create-replication-group.html, it ended up propagating the tags to the nodes in the replication group! I could also not retrieve those tags using list tag api:        https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/elasticache.html#ElastiCache.Client.list_tags_for_resource. 
Tested and found that it doesn't support resource group tagging atm

Reply:
Thanks for checking, it’s a little unclear on the resource group tagging support page

Reply:
Yep!! I tried calling the api, and got a response back about an invalid arn! In the docs page, I could only find support for cluster/snapshot arns